ocs_ci/helpers/sanity_helpers.py

```diff
@@ -138,6 +94,11 @@ def delete_resources(self):
             pvc_obj.delete()
         for pvc_obj in self.pvc_objs:
             pvc_obj.ocp.wait_for_delete(pvc_obj.name)
+        for obc_obj in self.obc_objs:
+            obc_obj[0].delete()
+            assert not self.ceph_cluster._mcg_obj.s3_verify_bucket_exists(
```

Comment:
This method might not work, since OC/CLI buckets have different CR names when compared to the bucket name itself.
The right way will be to use the object's `verify_deletion()` method -
```suggestion
            assert obc_obj.verify_deletion()
```

If it's found, the error is logged from within the function itself.

Reply:
+1